objectivec/ProtocolBuffers_OSX.xcodeproj/project.pbxproj

```diff
@@ -826,7 +828,7 @@
 				GCC_WARN_UNUSED_PARAMETER = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
 				GENERATE_PROFILING_CODE = NO;
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				MACOSX_DEPLOYMENT_TARGET = 10.10;
```

Comment:
Yeah, it's because of the swift test, as using a lower value includes the arclite library twice (once from the default toolchain and once from the swift 2.3 toolchain) and errors with duplicate symbols... Although I would expect this to be a bug in the xcode beta... should we wait for new releases to see if it gets fixed?